relay-sampling/src/lib.rs

```diff
@@ -1,4 +1,52 @@
-//! Functionality for calculating if a trace should be processed or dropped.
+//! Sampling logic for performing sampling decisions of incoming events.
+//!
+//! Relay performs sampling of events in order to allow Sentry to offer performance at scale. The
+//! process of sampling ultimately comes down to generating a random number and given a sample rate
+//! checking whether the generated number is bigger or smaller than the sample rate.
```

Comment:
I did not insert correct links, but

```suggestion
//! In order to allow Sentry to offer performance at scale, Relay extracts key [`metrics`] from all transactions, but only forwards a random sample of raw transaction payloads to the upstream. What exact percentage is sampled is determined by [`dynamic sampling rules`], and depends on the project, the environment, the transaction name, etc.
```